### DISCUSSION TITLE
PATCH method on SCIM API on fido2devices

### DISCUSSION THREAD

**Discussion by aguskwee** [JanssenProject/jans#8594] [General]
Hi, I want to be able to change the fido2 device status from registered to suspended / canceled and vise versa using fido2 patch method.

However, it seems that patch method has not been implemented yet. Is there a way in the roadmap to implement the PATCH method? Otherwise, is there any another way to update the fido2 devices through api?

Thanks, Agus

**Comment by jgomer2001** [comment]
I'm not aware of suspended / canceled statuses, I only see registered or pending:
https://github.com/JanssenProject/jans/blob/main/jans-scim/server/src/main/resources/jans-scim-openapi.yaml#L2506

In general, for fido2 device updates, use PUT, see

https://github.com/JanssenProject/jans/blob/main/jans-scim/server/src/main/resources/jans-scim-openapi.yaml#961

**Reply by ossdhaval** [reply]
Hi @aguskwee Hope @jgomer2001's comment addresses your concern. If so, can you kindly accept it as an answer and we can close this discussion.
